SessionMessagingKit/Shared Models/MessageViewModel.swift

```diff
@@ -715,54 +731,38 @@ public extension MessageViewModel {
                 FROM \(Interaction.self)
                 JOIN \(SessionThread.self) ON \(thread[.id]) = \(interaction[.threadId])
                 LEFT JOIN \(Contact.self) ON \(contact[.id]) = \(interaction[.threadId])
-                LEFT JOIN \(Profile.self) AS \(threadProfileTableLiteral) ON \(threadProfileTableLiteral).\(profileIdColumnLiteral) = \(interaction[.threadId])
+                LEFT JOIN \(Profile.self) AS \(threadProfile) ON \(threadProfile).\(profileIdColumn) = \(interaction[.threadId])
                 LEFT JOIN \(DisappearingMessagesConfiguration.self) ON \(disappearingMessagesConfig[.threadId]) = \(interaction[.threadId])
                 LEFT JOIN \(OpenGroup.self) ON \(openGroup[.threadId]) = \(interaction[.threadId])
                 LEFT JOIN \(Profile.self) ON \(profile[.id]) = \(interaction[.authorId])
-                LEFT JOIN (
-                    SELECT \(quote[.interactionId]),
-                           \(quote[.authorId]),
-                           \(quote[.timestampMs]),
-                           \(interaction[.body]) AS \(Quote.Columns.body),
-                           \(interactionAttachment[.attachmentId]) AS \(Quote.Columns.attachmentId)
-                    FROM \(Quote.self)
-                    LEFT JOIN \(Interaction.self) ON (
-                        (
-                            \(quote[.authorId]) = \(interaction[.authorId]) OR (
-                                \(quote[.authorId]) = \(blindedPublicKey ?? "") AND
-                                \(userPublicKey) = \(interaction[.authorId])
-                            )
-                        ) AND
-                        \(quote[.timestampMs]) = \(interaction[.timestampMs])
+                LEFT JOIN \(Quote.self) ON \(quote[.interactionId]) = \(interaction[.id])
+                LEFT JOIN \(Interaction.self) AS \(quoteInteraction) ON (
+                    \(quoteInteraction).\(timestampMsColumn) = \(quote[.timestampMs]) AND (
+                        \(quoteInteraction).\(authorIdColumn) = \(quote[.authorId]) OR (
+                            \(quoteInteraction).\(authorIdColumn) = '' AND
```

Comment:
Do we need to consider current user's `blindedPublicKey` here? Like `\(quoteInteraction).\(authorIdColumn) = \(blindedPublicKey ?? '')`

Reply:
Ah yep, must have mixed it up when copying between the DB browser and just left `authorId = ''` instead of `authorId = \(blindedPublicKey ?? '')` 🤦